projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
+	<!-- refactoring activeProjectDetail: Infosection with key actors, timelines etc-->
+	<div class="rtProjectInfoSection">
+	<section class="factorWrapper">
+	<div class="section group factors">
+		<div class="col span_3_of_12">
```

Comment:
Lets use [Bootstrap column classes](http://getbootstrap.com/css/#grid) here, so we have less CSS to maintain.